preprocess_utilities.py

```diff
@@ -67,3 +67,12 @@ def add_bipolar_derivation(raw, ch_1, ch_2):
     adds a channel to the given raw instance that is ch_1-ch_2
     """
     mne.set_bipolar_reference(raw, ch_1, ch_2)
+
+
+def process_epochs(trigger, epochs, notch_list=[50], highFilter=30, lowFilter=1, samp_rate=2048):
```

Comment:
Add documentation - what is the functions purpose? what type of data is it expecting to get? What is the meaning of each argument passed to the function?

Reply:
AHLA